src/meltano/core/venv_service.py

```diff
@@ -227,7 +227,10 @@ async def create(self) -> Process:
         """
         logger.debug(f"Creating virtual environment for '{self.namespace}/{self.name}'")
         try:
-            return await exec_async(sys.executable, "-m", "venv", str(self.venv))
+            return await exec_async(
+                os.path.join(os.path.dirname(sys.executable), "virtualenv"),
```

Comment:
I see you pushed a commit that switched this over to using `pathlib`, and then reverted it. Why?

Reply:
Pushed it without testing locally. Had to cast the `Path` instance to a string.